lib/utils/pyhelp.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * This file is part of the Micro Python project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 Colin Hogben
```

Comment:
That's why I kept Damien's copyright in pyhelp.c since that was just a minor reshuffle of the original help.c implementations.  What is your preference for new but trivial code such as pyhelp.h?